src/conf_mode/vpp_sflow.py

```diff
@@ -52,31 +58,32 @@ def get_config(config=None) -> dict:
         key_mangling=('-', '_'),
         get_first_key=True,
         no_tag_node_value_mangle=True,
+        with_recursive_defaults=True,
     )
 
     if system_sflow:
         config['system_sflow'] = system_sflow
 
-    if not config:
+    if effective_config:
+        config.update({'effective': effective_config})
+
+    if not conf.exists(base):
         config['remove'] = True
         return config
 
     # Add list of VPP interfaces to the config
     config.update({'vpp_ifaces': cli_ifaces_list(conf)})
 
-    if effective_config:
-        config.update({'effective': effective_config})
-
     return config
 
 
 def verify(config):
-    if 'remove' in config:
+    if 'remove' in config or 'remove_vpp' in config:
```

Comment:
`'remove_vpp' in config` will skip the whole verify section, but non-VPP interfaces and the daemon may still be configured.